viewing_party/main.py

```diff
@@ -0,0 +1,210 @@
+#Test_wave_1
+#checking four tests in create_movie function 
+def create_movie(title, genre, rating):
+    """
+    checks if three given parameter is True or False 
+    input: Three parameters(title, genre, rating)
+    output: dict values or None 
+    """
+
+    if title and genre and rating:
+        movies = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+        return movies
+    else:
+        return None 
+
+#checks one test 
+def add_to_watched(user_data, movie):
+    """
+    Adds dict to "watched"
+    input: Two parameters(user_data, movie)
+    output: updates data 
+    """
+    user_data["watched"].append(movie)
+    return user_data
+
+#checks one test 
+def add_to_watchlist(user_data, movie):
+    """
+    Adds dict to watchlist 
+    input: two parameters(user_data, movie)
+    output: updates data 
+    """
+    user_data["watchlist"].append(movie)
+    return user_data
+
+#There are three tests in watch_movie function 
+def watch_movie(user_data, movie):
+    """
+    stores data from "watchlist" to "watched"
+    input: two paramemters(user_data, movie)
+    output: returns updated data  
+    """
+    
+    for element in user_data["watchlist"]:
+        if movie == element["title"]:
+
+            user_data["watched"].append(element)

```

Comment:
Consider using `add_to_watched`
```suggestion
            add_to_watched(user_data, each_movie)
```